### PR TITLE
Created a styled template for repair details page

### PR DESCRIPTION
**Pull request by deberny**
Repair details page template:
localhost_3000_properties_000001_repairs_1 1

### CODE REVIEW

**Comment by deberny** [comment]
Might need some tweaking later, but I think it's okeish for now :)

**Comment by deberny** [comment]
[Build styled template for a single repair detailed view](https://trello.com/c/2dWgY3kl/29-build-styled-template-for-a-single-repair-detailed-view)

File: app/assets/stylesheets/_main_rcc.scss [hunk 1]
```diff
@@ -69,3 +64,148 @@
...
+/*
+ * Repair card
+ */
+
+.card {
+  border-radius: 2px;
+
+  &.status {
+    border-top: 8px solid $dark-gray;
+    &-completed {
```
Comment (james) [inline app/assets/stylesheets/_main_rcc.scss]: oo, didn't know you could do &- in sass, nice :-)

**Comment by deberny** [comment]
Thanks 😄

**Pull request merged**